framework/qlf/dashboard/bokeh/qaxwsigma/main.py

```diff
@@ -55,6 +57,9 @@
 snr       = metrics['snr']
 
 xwsigma = ast.literal_eval(xwsigma)
```

Comment:
We changed the database so there is no need to use `ast.literal_eval` anymore anywhere.